samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -433,15 +433,31 @@ CHIP_ERROR BLEConnectivityManager::Init(const bt_uuid **serviceUuids, uint8_t se
 	}
 #endif /* CONFIG_BT_SMP */
 
+	struct bt_le_scan_param scan_param = {
+		.type = BT_LE_SCAN_TYPE_PASSIVE,
+		.options = BT_LE_SCAN_OPT_FILTER_DUPLICATE,
+		.interval = CONFIG_BRIDGE_BT_SCAN_INTERVAL,
+		.window = CONFIG_BRIDGE_BT_SCAN_WINDOW,
+	};
+
+	struct bt_le_conn_param conn_param_ = {
```

Comment:
Additional underscore?
```suggestion
	struct bt_le_conn_param conn_param = {
```